source/lib/predicates/predicate.ts

```diff
@@ -54,17 +55,17 @@ export class Predicate<T = any> implements BasePredicate<T> {
 	 * @hidden
 	 */
 	// tslint:disable completed-docs
-	[testSymbol](value: T, main: Ow) {
-		const label = this.context.label
-			? `${this.type} ${this.context.label}`
+	[testSymbol](value: T, main: Ow, label?: string) {
+		const lbl = label
```

Comment:
I would rather call it `label2` than shortening it like this. Why not just overwrite the `label` argument?

Reply:
We enforce this with ts-lint -> `no-parameter-reassignment`.

Reply:
Ah ok